rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -629,74 +624,74 @@ public Long getRetiringFederationCreationTime(Object[] args)
         return creationTime.toEpochMilli();
     }
 
-    public Long createFederation(Object[] args)
+    public Integer createFederation(Object[] args)
     {
         logger.trace("createFederation");
 
-        int numberOfSignaturesRequired = ((BigInteger) args[0]).intValue();
-        try {
-            return bridgeSupport.createFederation(numberOfSignaturesRequired);
-        } catch (IOException e) {
-            logger.warn("Exception in createFederation", e);
-            throw new RuntimeException("Exception in createFederation", e);
-        }
-
+        return bridgeSupport.voteFederationChange(
+                rskTx,
+                new ABICallSpec("create", new byte[][]{})
+        );
     }
 
     public Integer addFederatorPublicKey(Object[] args)
     {
         logger.trace("addFederatorPublicKey");
 
-        byte[] publicKeyBytes = (byte[]) args[0];
-        BtcECKey publicKey;
+        byte[] publicKeyBytes;
         try {
-            publicKey = BtcECKey.fromPublicOnly(publicKeyBytes);
+            publicKeyBytes = (byte[]) args[0];
         } catch (Exception e) {
-            throw new BridgeIllegalArgumentException("Public key could not be parsed " + Hex.toHexString(publicKeyBytes), e);
+            logger.warn("Exception in addFederatorPublicKey: {}", e.getMessage());
```

Comment:
We will probably prefer to log the whole exception and not only the message in case we need to debug it.

Reply:
:+1: